docs/js/getting-started.md

```diff
@@ -20,53 +20,296 @@ import LicenseBadge from '../../src/sap/sdk-js/LicenseBadge'
 <> <BuildBadge /> { } <LicenseBadge /> </>
 
 ## Installation
+
 To install the CLI globally, run:
 
-```bash
+```shell
 $ npm install -g @sap-cloud-sdk/cli
 ```
 
 ### Update
+
 As long as the CLI version is less than `1.0.0`, run the following to update to the latest version. Please keep in mind, that these updates can have breaking changes as per the [semver spec](https://semver.org/#spec-item-4).
 
-```bash
+```shell
 $ npm install -g @sap-cloud-sdk/cli@latest
 ```
 
-## Usage
+## CLI Usage
 
-<!-- usage -->
-```bash
-$ npm install -g @sap-cloud-sdk/cli
-$ sap-cloud-sdk COMMAND
-running command...
-$ sap-cloud-sdk (-v|--version|version)
-@sap-cloud-sdk/cli/0.1.8 darwin-x64 node-v13.11.0
-$ sap-cloud-sdk --help [COMMAND]
-USAGE
-  $ sap-cloud-sdk COMMAND
-...
-```
-<!-- usagestop -->
-
-The CLI can initialize an nest-based project or (the more common case) add everything you need to develop for SAP Cloud Platform to an existing project no matter what backend framework you use.
+The CLI can initialize a nest-based project or (the more common case) add everything you need to develop for SAP Cloud Platform to an existing project no matter what backend framework you use.
 If there are any incompatibilities, please let us know in the [issues](https://github.com/SAP/cloud-sdk-cli/issues/new/choose)!
 
-To get started run
-```bash
+### Installation
+
+First run this command to install the CLI
+
+```shell
+npm install -g @sap-cloud-sdk/cli
+```
+
+### Usage
+
+```shell
+sap-cloud-sdk [COMMAND]
+```
+To get the CLI's version, run:
+```shell
+sap-cloud-sdk (-v|--version|version)
+```
+To get a list of all commands, run:
+```shell
+$ sap-cloud-sdk --help
+```
+
+### Create a new project
+
+To create a new project run the CLI's init command in the project folder.
+
+```shell
 $ sap-cloud-sdk init
 ```
-in the project folder.
 
 It will guide you through the initialization, create the necessary files and add necessary dependencies.
 If you run it in an empty folder, it will ask if you want to initialize a project using [@nest/cli](https://github.com/nestjs/nest-cli).
 
-To deploy to and run on Cloud Foundry, you need to
-1. Make sure that your app listens to port 8080
-2. Build your app if necessary
-3. Run [`sap-cloud-sdk package`](#sap-cloud-sdk-package)
-4. Push to Cloud Foundry (`cf push`)
+The CLI will already install all the necessary dependencies for the project, so this might take a minute. If everything worked correctly, you should see output like this:
```

Comment:
```suggestion
The CLI will already install all the necessary dependencies for the project, so this might take a minute. If everything worked correctly, you should see the following output:
```